app/src/main/kotlin/compiler/syntax/LanguageGrammar.kt

```diff
@@ -32,13 +32,21 @@ object LanguageGrammar {
         val callArgs1 = getProduction("CALL_ARGS", "({nE_EXPR}({tCOMMA}$NLS{nE_EXPR})*({tCOMMA}$NLS{nE_EXPR}{tASSIGNMENT}$NLS{nE_EXPR})*)?")
         val callArgs2 = getProduction("CALL_ARGS", "{nE_EXPR}{tASSIGNMENT}$NLS{nE_EXPR}({tCOMMA}$NLS{nE_EXPR}{tASSIGNMENT}$NLS{nE_EXPR})*")
 
+        // TODO: add "wewnątrz" handling.
```

Comment:
The merge is going to be nontrivial, so I think that we should come up with a clever way of doing it.

Reply:
You can assume that there's some method for making a loop which iterates over an array.
https://github.com/TCS-Compilers/compiler/pull/178/files#r1083533608